inst/doc/shared_cache.Rmd

```diff
@@ -1,52 +1,191 @@
 
 ### Guidelines
 
-Projects using a shared cache should follow these guidelines:
+####Projects using a shared cache should follow these guidelines:
+
+* Use <ind>**indicator files**</ind> (usually an `.ind` suffix) to represent most or all of the chain of connected <target>**targets**</target> within your remake files. Each `.ind` file should be one of two products of a recipe (e.g. R function call), where the other product is the creation of a <data>**data file**</data>, either locally and/or in the shared cache. The <ind>**indicator files**</ind> are the only products that are declared to `remake`, while the <data>**data files**</data> remain hidden. This allows the <ind>**indicator files**</ind> to indirectly represent the <data>**data files**</data> to `remake` thereby enabling compatibility between `remake` and the shared cache. 
+
+* Always build <target>**targets**</target> using `scipiper::scmake()` rather than `remake::make()`. Though the functions are outwardly very similar, `scmake()` maintains an extra layer of metadata that allows multiple users to share a single project <build>**build status**</build> (e.g., "file x.rds.ind is up to date; file y.rds.ind is out of date"). In a shared-cache project, you should not even need to load the `remake` package directly.
+
+* Generally avoid using R objects as `remake/scmake` <target>**targets**</target>...but if you must, usually for convenience or conciseness of the workflow plan, recognize that R objects must be built by every analyst. So if a target takes a non-trivial length of time to build, or if it depends on large volumes of <data>**data**</data> as input, that target should usually be a file rather than an R object.
+
+* `git commit` all <ind>**indicator files**</ind> and <build>**build files**</build> (with occasional exception of <ind>**indicator files**</ind> within a task plan; those require additional thought). `git ignore` all <data>**data files**</data> unless they are small enough to store in git/GitHub, such as small, text-based, and typically hand-curated <data>**data files**</data> (e.g. a <data>**data file**</data> that matches NHD lake ID's to collaborator-provided <data>**data files**</data>). <data>**Data files**</data> that are  necessary to pipeline basic functions should also be committed (e.g. <data>**gd_config.yml**</data>). 
+
+* To force a rebuild, either use the `force=TRUE` argument to `scmake()` or use `scdel()` to delete <ind>**indicator files**</ind>. There's seldom any benefit to deleting <data>**data files**</data> (by any method); usually deleting the <ind>**indicator files**</ind> is plenty. When deleting <ind>**indicator files**</ind>, `force=TRUE` or `scdel()` are preferable to directly deleting the `.ind` files because if only the `.ind` files are deleted, the `scipiper` database may fail to update properly when the `.ind` files are rebuilt.
```

Comment:
Should we mention anything about touching (or most likely not touching) the `remake/build` files?